WORKING-GROUPS.md

```diff
@@ -61,13 +61,13 @@ Document Folder | [Folder](https://drive.google.com/corp/drive/u/0/folders/0BzW5
 
 ## Config
 
-Config API, Config management/rollout
+Config and config management
```

Comment:
Config and config [...] stutters a bit no ?

Reply:
It's fixed. thanks